packages/manager/modules/pci/src/projects/project/notebooks/notebooks.service.js

```diff
@@ -0,0 +1,103 @@
+export default class NotebookService {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  static getIcebergHeaders() {
+    return {
+      headers: {
+        'X-Pagination-Mode': 'CachedObjectList-Pages',
+        'X-Pagination-Size': 50000,
```

Comment:
pagination-size is required?

Reply:
Yes... Else, only the first 10 records are returned